packages/core/tocco-ui/src/Select/Select.js

```diff
@@ -80,11 +84,10 @@ const Select = ({
 
   const handleFocus = () => {
     selectComponent.current.focus()
+    setWrapperWidth(getWrapperWidth)
+    setWrapperHeight(getWrapperHeight)
```

Comment:
`setState` function do allow to pass in a function instead of a value. However the idea behind is, that you can access the `prevState` as a param:
```
setState((prev) => [...prev, addedItem]);
```
Therefore I would suggest to just pass in a value here for simplicity reasons: `setWrapperHeight(getWrapperHeight())`